packages/pieces/community/file-helper/src/lib/actions/convert-file.ts

```diff
@@ -0,0 +1,46 @@
+import { Property, createAction } from '@activepieces/pieces-framework';
+import mime from 'mime-types';
+import { encodings } from '../common/encodings';
+
+export const convertFileAction = createAction({
```

Comment:
Please rename this action to changeFileEncoding

Reply:
Done: https://github.com/activepieces/activepieces/pull/6098/files#diff-4e48b12c0cd432e3012da325a9ef727f529a3665b127c0aaa9a62f28afb02abfR5